packages/pythonlib/pythonlib.v0.15.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "Jane Street developers"
+authors: ["Jane Street Group, LLC"]
+homepage: "https://github.com/janestreet/pythonlib"
+bug-reports: "https://github.com/janestreet/pythonlib/issues"
+dev-repo: "git+https://github.com/janestreet/pythonlib.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/pythonlib/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"      {>= "4.11.0"}
```

Comment:
```suggestion
  "ocaml"      {>= "4.14.0"}
```

Reply:
Actually maybe i could’ve used `[%%if ocaml_version ...` in my PR. Could you add those where necessary if you think this is needed? (in the current state this release skips OCaml 4.13)

Reply:
Nevermind, I figured it out.